scripts/setup.py

```diff
@@ -78,13 +78,62 @@ def download_and_install_package(url_to_retrieve, filename):
             downloaded.
         filename: string. The name of the tar file.
     """
+
```

Comment:
Don't need to introduce this newline. Keep things consistent with the existing file.

Reply:
Removed